app/javascript/tests/patient/monitoring_actions/CaseStatus.test.js

```diff
@@ -175,6 +176,15 @@ describe('CaseStatus', () => {
     expect(wrapper.state('monitoring')).toBeFalsy();
     expect(wrapper.find(Button).at(1).prop('disabled')).toBeFalsy();
     expect(wrapper.find('p').at(1).text()).toEqual('The case status for the selected record will be updated to Confirmed and moved to the closed line list in the current workflow.');
+    expect(wrapper.find('ModalBody').find('p').at(0).text()).toContain('Please select what you would like to do:');
```

Comment:
Can you add check for if this exists or not when the other dropdown items are changed?

Reply:
fixed